pkg/routes/favorite_quickstart_test.go

```diff
@@ -0,0 +1,104 @@
+package routes
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/RedHatInsights/quickstarts/pkg/database"
+	"github.com/RedHatInsights/quickstarts/pkg/models"
+	"github.com/go-chi/chi/v5"
+	"github.com/stretchr/testify/assert"
+)
+
+var accountTestId = "testAccountID"
+var allTestIdFavorites []models.FavoriteQuickstart
+
+func mockQuickstartFavoritable(qsName string) *models.FavoriteQuickstart {
+	var qs models.Quickstart
+	qs.Name = qsName
+
+	favQuickstart := models.FavoriteQuickstart{
+		AccountId:      accountTestId,
+		QuickstartName: qs.Name,
+		Favorite:       true,
+	}
+	qs.FavoriteQuickstart = append(qs.FavoriteQuickstart, favQuickstart)
+
+	database.DB.Create(&qs)
+
+	allTestIdFavorites = append(allTestIdFavorites, favQuickstart)
+	return &favQuickstart
+}
+
+func setupFavoriteQuickstartRouter() *chi.Mux {
+	r := chi.NewRouter()
+	r.Get("/", GetAllFavoriteQuickstarts)
+	r.Post("/", UpdateFavoriteQuickstart)
+	return r
+}
+
+func TestFavoriteQuickstarts(t *testing.T) {
```

Comment:
These look great. I think we should add some testing for error conditions to make sure the `handleError` method is tested